troubleshooting/known_issues/sat_bootprep_image_customization_error.md

```diff
@@ -0,0 +1,44 @@
+# Known Issue: `sat bootprep` image customization error

```

Comment:
Remove the leading "Known issue:" to match consistency with other known issue headers. This also makes the table-of-content entry match the level-1 header.
```suggestion
# `sat bootprep` image customization error

```

Reply:
Addressed